src/classes/Pantry.js

```diff
@@ -1,46 +1,69 @@
+const { ingredientsData } = require("../data/ingredients");
 const { userData } = require("../data/users");
 
 class Pantry {
   constructor(user) {
     this.shelf = user.pantry;
   }
 
-  // determineAmountOfIngredientsUserHas(recipe) {
-  //   let userIngredientsAmt = this.shelf.reduce((allUserIngredientAmt, userIngredient) => {
-  //     recipe.ingredients.forEach(recipeIngredient => {
-  //       if (recipeIngredient.id === userIngredient.ingredient)
-  //         allUserIngredientAmt.push(userIngredient.amount)
-  //     })
-  //     return allUserIngredientAmt;
-  //   }, [])
-  //   return userIngredientsAmt
-  // }
-
-  determineAmountOfIngredientsUserHas() {
-    let userIngredientAmt = this.shelf.map(ingredient => {
-      return ingredient.amount 
+  showPantryIngredientInfo() {
```

Comment:
Pushing in the information in the form of an object was a great idea!

Reply:
Heck yea!